src/control/server/engine/config_test.go

```diff
@@ -581,15 +590,64 @@ func TestConfig_ToCmdVals(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if diff := cmp.Diff(wantArgs, gotArgs, cmpOpts()...); diff != "" {
+	if diff := cmp.Diff(wantArgs, gotArgs, defConfigCmpOpts...); diff != "" {
 		t.Fatalf("(-want, +got):\n%s", diff)
 	}
 
 	gotEnv, err := cfg.CmdLineEnv()
 	if err != nil {
 		t.Fatal(err)
 	}
-	if diff := cmp.Diff(wantEnv, gotEnv, cmpOpts()...); diff != "" {
+	if diff := cmp.Diff(wantEnv, gotEnv, defConfigCmpOpts...); diff != "" {
 		t.Fatalf("(-want, +got):\n%s", diff)
 	}
 }
+
+func TestConfig_setAffinity(t *testing.T) {
+	for name, tc := range map[string]struct {
+		cfg     *Config
+		expErr  error
+		expNuma uint
+	}{
+		"numa pinned; matching iface": {
+			cfg: MockConfig().
+				WithPinnedNumaNode(1).
+				WithFabricInterface("ib1"),
+			expNuma: 1,
+		},
+		// NOTE: this currently logs an error but could instead return one
+		//       but there might be legitimate use cases e.g. sharing interface
+		//"numa pinned; not matching iface": {
```

Comment:
Intend to re-enable the test? With whatever the current behavior is.

Reply:
wasn't planning on it as this is more of a note to indicate that we could change in future if we wanted to be less conservative but I can remove it instead